pkg/upload/all.go

```diff
@@ -20,18 +20,19 @@ func All(
 	overwrite bool,
 	apiKey string,
 	dryRun bool,
+	logger log.Logger,
 ) error {
 
 	// Build the file list from the path(s)
-	log.Info("building file list...")
+	logger.Info("building file list...")
 
 	fileList, err := utils.BuildFileList(paths)
 
 	if err != nil {
-		log.Error(" error building file list", 1)
+		logger.Fatal(" error building file list")
```

Comment:
We should also include the error message

```suggestion
		logger.Fatal("Error building file list")
```